jrank/gen_api_model_answer.py

```diff
@@ -6,28 +6,31 @@
 python3 gen_api_answer.py --model claude-2 --bench-name rakuda_v2_test
```

Comment:
line 87 is updated to benchmark, to change `--bench-name` to `benchmark`.
also change rakuda_v2_test?